server/src/internalClusterTest/java/org/opensearch/snapshots/RestoreSnapshotIT.java

```diff
@@ -282,6 +282,8 @@ public void testRestoreOperationsShallowCopyEnabled(boolean remoteTranslogEnable
         ensureYellowAndNoInitializingShards(restoredIndexName1);
         ensureGreen(restoredIndexName1);
         assertDocsPresentInIndex(client(), restoredIndexName1, numDocsInIndex1);
+        // Re-initialize client to make sure we are not using client from stopped node.
+        client = client();
```

Comment:
https://github.com/opensearch-project/OpenSearch/blob/a3b515aaa7e14a18f98aef80658a630bb62bb4cc/test/framework/src/main/java/org/opensearch/test/InternalTestCluster.java#L861-L865

I skimmed a bit through the InternalTestCluster and i could not be sure if it would return a living node. We can point the client to cluster manager to be sure.